### DISCUSSION TITLE
Will support nautobot in the future?

### DISCUSSION THREAD

**Discussion by xth0331** [bb-Ricardo/netbox-sync#87] [General]
Nautobot was initially developed as a fork of NetBox, I see some plugins for netbox move to Nautobot app platform.

**Comment by bb-Ricardo** [comment]
Hi, Thank you for this idea. For some reason I completely missed the forking of NetBox.

I guess right now it should be quite easy to support Neutobot but can get quite difficult in the future as these two projects diverge more and more.

First I need to setup a Nautobot to test compatibility.

Are you using Nautobot? What are your experiences?

**Comment by bb-Ricardo** [comment]
NetBox 3.0 just got release which also adds GraphQL support and a fancy new UI.

I tested Nautobot and except a small change it seems to work.

Is this feature still required? Would rather not support two different tools which slowly divert in their feature set.

**Reply by bb-Ricardo** [reply]
I guess, Nautobot is not required anymore.
